frontend/src/components/FAQs.tsx

```diff
@@ -0,0 +1,342 @@
+import plusIcon from "../assets/plusIcon.svg";
+import { Accordion } from "../components/index.ts";
+import styles from "../stylesheets/Home.module.css";
+
+import { useState } from "react";
+
+export function FAQs() {
+  const [expandAllCertificationInfo, setExpandAllCertificationInfo] = useState(false);
+  const [expandAllPathwaysInfo, setExpandAllPathwaysInfo] = useState(false);
+
+  const toggleExpandAllCertificationInfo = () => {
+    setExpandAllCertificationInfo(!expandAllCertificationInfo);
+  };
+  const toggleExpandAllPathwaysInfo = () => {
+    setExpandAllPathwaysInfo(!expandAllPathwaysInfo);
+  };
+
+  return (
+    <>
+      <div className={styles.textDivider2}>
+        <div className={styles.heading}>FAQ's</div>
+      </div>
+
+      <div className={styles.FaqsContainer}>
+        <div className={styles.infoTitle}>
+          <h2 className={styles.red}>Certification Information</h2>
+          <button className={styles.expand} onClick={toggleExpandAllCertificationInfo}>
+            <span style={{ marginLeft: 20 }}>
```

Comment:
Since we'll have a css file for this componenet lets try to keep the css in the css file and not in the tsx file.